src/ResourceManager/Network/Commands.Network/ExpressRouteCircuit/Peering/NewAzureExpressRouteCircuitPeeringConfigCommand.cs

```diff
@@ -72,4 +74,10 @@ public override void Execute()
             WriteObject(peering);
         }
     }
+
+    public enum LegacyMode
```

Comment:
@dihan0604 please use the link to update help for the new parameter
https://github.com/Azure/azure-powershell/blob/preview/documentation/help-generation.md